FlightPlanner/FlightPlanner.Core/Dto/SearchFlightResponse.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace FlightPlanner.Core.Dto
+{
+
+    public class SearchFlightResponse
+    {
+
+        private readonly static object balanceLock = new object();
+
+        [Required]
+        [JsonPropertyName("items")]
```

Comment:
vai šeit to json property name vajag? man liekas ka viņš prot camelCase pats atserializēt un saserializēt.

Reply:
Nevajag, atstāju katram gadījuamm, lai paliek atmiņā, ja nu vajag